participants/fmusic for slingshot.md

```diff
@@ -0,0 +1,60 @@
+###(0) Project Name and Team Slack Handles
+- Our project Name： FMusic
+- Team Slack Handles：[FUNBLOCKS](https://funbloks.slack.com/join/shared_invite/zt-hy0c64sg-pGNuL92B~azTpyZz8M6jgg#/)
+
+###(1) Brief Project Description
+- FMusic is a global  music source shared system. 
+- Users can upload the music they like and the music they want to collect.
+- After uploading to FMusic, they can download it at any time and share it with others. 
+- You can also enjoy shared music files uploaded by other users.
+- We aim to establish an open music community to prevent centralized monopoly.
+
+###(2) Link to Project UI
+- Link : https://axhub.im/ax9/23d60b55baa41e75/#g=1
+
+###(3) What does your application/UI do?
+- FMusic is a music sharing platform.
+- FMusic provides users with an entrance to upload music files, after  user uploads their music files to FMusic, FMusic is responsible for the on-chain interaction with the Filecoin network and permanently stores the user’s files on the Filecoin network.
+- When they needs to find the music resources they wants, Fmusic can also find all the music files that have been uploaded to the Filecoin network and provide users to download them. 
+- When users need to save important music files, we will charge the corresponding FIL as a handling fee based on the gas fee and duration of the transfer on the chain.
+- Similarly, users also need to pay a certain fee for downloading to get what they wants.
+
+###(4) If your project is using a curated dataset, which one are you using?
+- We use the official data set: Free Music Archive :https://github.com/mdeff/fma
+
+###(5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+- We are the official data set used.
+
+###(6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+- We planing to upload about 300TB of data to the Filecoin network.
```

Comment:
I thought the Free Music Archive is less than 1 TB.  Where is the rest of the data volume coming from?
Ref:  https://github.com/mdeff/fma (879 GB)